process-deploy/src/main/java/com/wipro/ats/bdre/pdeploy/ProcessDeployMain.java

```diff
@@ -25,10 +25,9 @@
  */
 public class ProcessDeployMain {
 
-    private ProcessDeployMain(){}
-
     private static final Logger LOGGER = Logger.getLogger(ProcessDeployMain.class);
-
+    private ProcessDeployMain()
+    {}
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)